src/Data/User.elm

```diff
@@ -0,0 +1,41 @@
+module Data.User exposing
+    ( User
+    , decode
+    , encode
+    )
+
+import Json.Decode as Decode exposing (Decoder)
+import Json.Decode.Pipeline as Pipe
+import Json.Encode as Encode
+
+
+type alias User =
+    { email : String
+    , firstname : String
+    , lastname : String
+    , company : String
+    , cgu : Bool
+    , token : String
+    }
+
+
+decode : Decoder User
+decode =
+    Decode.succeed User
+        |> Pipe.required "email" Decode.string
+        |> Pipe.required "first_name" Decode.string
+        |> Pipe.required "last_name" Decode.string
+        |> Pipe.optional "organization" Decode.string ""
```

Comment:
This is a fix for the admin login bug, it's also fixed on python side by returning "" instead of null for the organization